website/static/js/tests/osfHelpers.test.js

```diff
@@ -264,6 +264,14 @@ describe('osfHelpers', () => {
         });
     });
 
+    describe('textareaElementDecode', () => {
+        it('returns properly decoded text', () => {
+            var text = '&amp; &lt; &gt;';
```

Comment:
What happens if `text = '&amp; &lt; &gt; & <>';`?

Reply:
Updated to...
```
      describe('textareaElementDecode', () => {
          it('returns properly decoded text', () => {
~             var text = '&amp; &lt; &gt; & < >';
              var ret = $osf.textareaElementDecode(text);
~             assert.equal(ret, '& < > & < >');
          });
      });
```
Which passes